openid-sse-framework-1_0.xml

```diff
@@ -904,157 +1127,308 @@ Cache-Control: no-store
     "urn:example:secevent:events:type_3"
   ]
 }
-]]></artwork>
-            </figure>
+]]></artwork>
+            </figure>
+
+            <t>Pending conditions or errors are signaled with HTTP status codes as follows:</t>
+
+            <texttable title="Replace Stream Configuration Errors" anchor="tabreplaceconfig">
+              <ttcol align="left">Code</ttcol>
+              <ttcol align="left">Description</ttcol>
+              <c>202</c>
+              <c>if the replace request has been accepted, but not processed. Receiver MAY try the
+                same request later in order to get processing result.</c>
+              <c>400</c>
+              <c>if the request body cannot be parsed, a read-only property is incorrect, or if the request is otherwise invalid</c>
+              <c>401</c>
+              <c>if authorization failed or it is missing</c>
+              <c>403</c>
+              <c>if the Event Receiver is not allowed to replace the stream configuration</c>
```

Comment:
s/allowed/authorized

Reply:
I'd prefer to leave these consistent with the other error definitions in the spec. We can create a new issue to discuss changing all of them at once.